event_listener/dfusion_db/management/commands/run_listener.py

```diff
@@ -14,4 +14,4 @@ def handle(self, *args: Any, **options: Any) -> None:
         event_listener = EventListener()
         while 1:
             event_listener.execute()
-            time.sleep(3)
+            time.sleep(2)
```

Comment:
![image](https://user-images.githubusercontent.com/1200333/61393503-664d3080-a8c1-11e9-9bcc-2d2af195f7ee.png)

Reply:
Damn that you were looking over the PR just right now. 

It was never my intention to keep this in here, just wanted to see quickly, whether it helps with the non-deterministic Travis test

🗡